bolognese.gemspec

```diff
@@ -50,6 +50,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'simplecov', '0.17.1'
   s.add_development_dependency 'hashdiff', ['>= 1.0.0.beta1', '< 2.0.0']
   s.add_development_dependency 'byebug'
+  s.add_development_dependency 'json-canonicalization', '0.3.1'
```

Comment:
we already have this gem added in gemspec in master https://github.com/datacite/bolognese/blob/master/bolognese.gemspec#L53